docker-compose.yml

```diff
@@ -568,7 +714,15 @@ services:
       - asset-manager
       - diet-error-handler
     environment:
+      RAILS_ENV: production
+      GOVUK_APP_DOMAIN: dev.gov.uk
+      GOVUK_WEBSITE_ROOT: http://www.dev.gov.uk
+      GOVUK_ASSET_ROOT: http://assets-origin.dev.gov.uk
+      GOVUK_ASSET_HOST: http://static.dev.gov.uk
+      SECRET_KEY_BASE: 875c6bf4c48da9bb41f4cfd25d09bf5e2a62d88b39efc4bd9c498e6c8f61e4df740af87386f97269525e01b5f74402512eb6a4723882579f10aa95f6e2971fc2
+      RAILS_SERVE_STATIC_FILES: "true"
       SENTRY_CURRENT_ENV: whitehall
+      DISABLE_DATABASE_ENVIRONMENT_CHECK: 1
```

Comment:
Do you know why this is needed for only one app?

Reply:
This is unfortunately related to the same oddity discussed https://github.com/alphagov/publishing-e2e-tests/pull/80/files#r154604899 here with Whitehall having an odd setup step.

Reply:
Sure, that makes sense. We should probably do some work to make them consistent - I'll put something on the backlog.

If it's just for that command do you think it might be more contextual having it inline like: https://github.com/alphagov/publishing-e2e-tests/pull/94/files#diff-b67911656ef5d18c4ae36cb6741b7965R47 ?